### DISCUSSION TITLE
How to paste in command mode or search mode？

### DISCUSSION THREAD

**Discussion by robin2014** [JetBrains/ideavim#484] [Q&A]
I can't paste in Clion(windows version) when enter command(type :) or search mode(type /). First i select some text and then right click mouse,then copy the text,then  enter search mode(type /),both ctrl v and right click mouse doesn't work?

**Comment by luooofan** [comment]
Try to use registers.
Help manual: [Vim: cmdline.txt](https://vimhelp.org/cmdline.txt.html#c_CTRL-R) (Or you can type `:help c_CTRL-R` in vim)

**Comment by sharpchen** [comment]
```vim
cnoremap <C-v> <C-r>+
```
